### DISCUSSION TITLE
[Request]: Sign Outするとメールアドレスの確認、利用規約の同意からやり直しになる。

### DISCUSSION THREAD

**Discussion by foriver4725** [iputapp/lounas#259] [雑談・リクエスト]
### バグの詳細

つまり新規登録時の操作を最初からやり直しになる。
ユーザー情報は保存されているもよう

### ブラウザの種類

Chrome

### ブラウザのバージョン

Chrome 117.0.5938.153

### 行動規範 (Code of Conduct)

- [X] このプロジェクトの行動規範に同意します。

**Comment by wiyco** [comment]
ご報告ありがとうございます！

> Sign Outするとメールアドレスの確認、利用規約の同意からやり直しになる。

現在こちらの動作は仕様になっております😭

なお、確認画面は正式版にて対応予定です。
何卒よろしくお願いします🙇
